playbooks/openstack-update_caasp.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- hosts: osh-deployer
+  gather_facts: no
+  tasks:
+    - name: Load variables
+      include_vars: "{{ item }}"
+      loop:
+        - "{{ playbook_dir }}/../vars/common-vars.yml"
+        - "{{ playbook_dir }}/../vars/manifest.yml"
+      tags:
+        - always
+
+    - name: Use Velum tooling to update CaaSP admin node
+      shell: "./velum-interactions --update-admin -e {{ upstream_repos_clone_folder }}/ccp/environment.json"
+      args:
+        executable: /bin/bash
+        chdir: "{{ upstream_repos_clone_folder }}/ccp/velum-bootstrap"
```

Comment:
Does this command waits for the admin to be restarted? or it just triggers the restart and exits?

Reply:
I think it waits for admin to be back online